scss/_variables.scss

```diff
@@ -1949,10 +1949,11 @@ $alert-border-width:                var(--#{$prefix}border-width) !default;
 
 // Boosted mod
 $alert-padding-sm:                  $spacer * .5 !default;
+// Know that using a map for the icon will avoid color change
```

Comment:
Not sure to understand the meaning of this comment, are you talking about the text color that can't be changed?
Maybe add in the comment because it's not obvious and is a bit of a hack: why do we have icons directly defined and why the warning one is special with this `("icon": xxxx)` structure?